docker/README.md

```diff
@@ -42,51 +42,39 @@ If you're more comfortable with mounting data to a data container, the commands
 
 ```
 # Create data container
-docker run --name=gogs-data --entrypoint /bin/true gogs/gogs
+docker run --name=gitea-data --entrypoint /bin/true gitea/gitea
 
 # Use `docker run` for the first time.
-docker run --name=gogs --volumes-from gogs-data -p 10022:22 -p 10080:3000 gogs/gogs
+docker run --name=gitea --volumes-from gitea-data -p 10022:22 -p 10080:3000 gitea/gitea
 ```
 
-#### Using Docker 1.9 Volume command
+#### Using Docker Volume command
 
 ```
 # Create docker volume.
-$ docker volume create --name gogs-data
+$ docker volume create --name gitea-data
```

Comment:
There's techincally no need for this, since `docker run` will auto-create it if it doesn't exist